src/util/announcement/oneSignalAnnouncement.ts

```diff
@@ -27,7 +27,8 @@ export function sendOneSignalNotification(text: string): Promise<any> {
     const message: Object = {
       app_id: process.env.ONE_SIGNAL_API_KEY,
       contents: {"en": text},
-      included_segments: ["All"]
+      headings: {"en": "StudentHack Hub"},
```

Comment:
Could we move this to an environment variable? Though I guess a better place would be to have some settings file which is separate from the environment variables but we can leave that for the future.

Reply:
Yes, good idea

Reply:
Changed in the latest commit